experimental/trusted_shuffler/scripts/run

```diff
@@ -2,15 +2,22 @@
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
 
+cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
 cargo build --manifest-path=experimental/trusted_shuffler/backend/Cargo.toml
- "${SCRIPTS_DIR}/run_backend" &
-readonly BACKEND_PID=$!
+cargo build --manifest-path=experimental/trusted_shuffler/client/Cargo.toml
 
-cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
-"${SCRIPTS_DIR}/run_server" &
+RUST_LOG=info cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
+  --k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888" &
```

Comment:
k=1 ?

Reply:
This PR doesn't include the possibility of sending more than 1 request yet

Reply:
The next PR will be about sending multiple requests from the test client (and checking the delay as well)